platform/how-to/uninstall.mdx

```diff
@@ -2,40 +2,95 @@
 title: Uninstall vCluster Platform
 sidebar_label: Uninstall vCluster Platform
 sidebar_position: 8
+description: Learn how to uninstall the vCluster Platform from your Kubernetes cluster using the vCluster CLI or Helm.
 ---
 
-## Delete Helm Release
+import VersionBadge from "@site/src/components/VersionBadge";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-```bash
-helm delete -n vcluster-platform loft
+import Prerequisites from '../_partials/install/base-prerequisites.mdx';
+
+
+This guide provides instructions on how to uninstall the platform from your Kubernetes cluster.
+
+:::warning
+This action is performed against the cluster the kube-context is pointing to, not the platform instance that is logged in. It does not require logging in to the platform.
+:::
+
+## Prerequisites
+
+<Prerequisites />
+
+## Uninstall
+
+There are two ways to uninstall the platform.
+
+<Tabs>
+    <TabItem value="Using vCluster CLI">
+<VersionBadge version="v4.2" vclusterVersion="v0.22" />
+
+### Delete the platform
+
+Note that the
+[VirtualClusterInstances](/platform/api/resources/virtualclusterinstance)
+CDRs managed with driver Helm are deleted, but the underlying virtual clusters
+are not uninstalled.
+
+On the Kubernetes cluster where the platform is installed, and all connected clusters, run:
+
+```bash title="Delete the platform using vCluster CLI"
+vcluster destroy platform
 ```
 
-Please make sure you also uninstall kiosk, if you have connected the loft cluster:
+:::info
+The command does the following:
+- Uninstalls Helm releases related to Loft.
+- Deletes CustomResourceDefinitions (CRDs).
```

Comment:
What CRDs does it delete? We don't have to list them out necessarily but it would be nice to clarify what CRDs (e.g. Deletes CRDs that were created by the platform).